frontend/src/pages/Workflows/reducer.js

```diff
@@ -160,7 +149,9 @@ const defaultState = fromJS({
   subprojectValidator: "",
   hasSubprojectValidator: false,
   fixedWorkflowitemType: "",
-  hasFixedWorkflowitemType: false
+  hasFixedWorkflowitemType: false,
+  rejectReason:"",
+  isRejectReasonDialogShown: undefined
```

Comment:
default for `isRejectReasonDialogShown` is false. Undefined should never be used for this because `is` implies a boolean var